js/runner.go

```diff
@@ -197,6 +197,11 @@ func (r *Runner) newVU(id int64, samplesOut chan<- stats.SampleContainer) (*VU,
 		MaxIdleConnsPerHost: int(r.Bundle.Options.BatchPerHost.Int64),
 	}
 	_ = http2.ConfigureTransport(transport)
+	http2Transport, err := http2.ConfigureTransports(transport)
```

Comment:
You shouldn't call both `http2.ConfigureTransport()` and `http2.ConfigureTransports()`, they do the same thing.

```suggestion
	http2Transport, err := http2.ConfigureTransports(transport)
```

Also, you may have to pass the `http2Transport` as the `VU.Transport` below, though I'm not completely sure about that... :man_shrugging: